src/thread/ThreadPreview.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+
+const HiddenThread: React.FC<HiddenThreadProps> = ({ 
+    threadId, 
+    boardId, 
+    threadHidden, 
+    setHideCallback 
+}) => {
+  return (
+    <div className="post hidden" key={threadId}>
+      This thread was hidden{" "}
+      <a
```

Comment:
This is not a must have, but we have the `ActionLink` component in this library, which is used to smartly render a button or an anchor according to some accessibility conditions: https://github.com/BobaBoard/boba-components/blob/b3b4c159107ab1dd0c278265134684914fb3e4a2/src/buttons/ActionLink.tsx#L134